cassandra/metadata.py

```diff
@@ -215,6 +216,20 @@ def _build_usertype(self, keyspace, usertype_row):
         return UserType(usertype_row['keyspace_name'], usertype_row['type_name'],
                         usertype_row['field_names'], type_classes)
 
+    def _add_table_metadata_to_ks(self, keyspace_metadata, row, col_rows, trigger_rows):
```

Comment:
Wondering if this functionality would be better implemented in ``KeyspaceMetadata``, where the index is held (KeyspaceMetadata.add_table(self, table_meta), or something like that).